JitsiConference.js

```diff
@@ -2530,6 +2553,57 @@ JitsiConference.prototype._onIceConnectionEstablished = function(
 
 };
 
+/**
+ * Called when the chat room reads a new list of properties from jicofo's
+ * presence. The properties may have changed, but they don't have to.
+ *
+ * @param {Object} properties - The properties keyed by the property name
+ * ('key').
+ * @private
+ */
+JitsiConference.prototype._propertiesChanged = function(properties) {
+    const changed = isEqual(properties, this.properties);
+
+    this.properties = properties;
+    if (changed) {
+        this.eventEmitter.emit(JitsiConferenceEvents.PROPERTIES_CHANGED);
```

Comment:
Timing wise, does it make a difference that this event is fired before analytics updates itself?

Reply:
Also, I noticed for JitsiConferenceEvents.SERVER_REGION_CHANGE I noticed you also emit out the new region but here you don't emit the properties. Is there a reason for the difference?

Reply:
It didn't seem good to emit the properties because they have some sort of structure that JitsiConference maintains. But now that I think about it, it's just a map, so I'll emit it.